requirements/constraints.txt

```diff
@@ -93,7 +93,7 @@ libsass==0.10.0
 click==8.1.6
 
 # pinning this version to avoid updates while the library is being developed
-openedx-learning==0.11.2
+openedx-learning @ git+https://github.com/open-craft/openedx-learning/@jill/collection-key
```

Comment:
TO DO: fix this dependency once https://github.com/openedx/openedx-learning/pull/223 merged & tagged